docs/storefront/store-management.md

```diff
@@ -6,11 +6,155 @@ Zustand allows us to create multiple stores. Each store consists of `slices`, wh
 
 ## Store structure
 
-Under folder `/project-base/storefront/store`, you can find all possible stores available for storing any kind of values.
+Under folder `/project-base/storefront/store`, you can find all possible stores available for storing any kind of values. Below, all stores are described, together with specific things for them, and when they should be used.
 
-These are stores which we currently use:
+## Persist store
 
--   **Persist Store** - Store which is persisted. That means that information saved in this store will persist after you reload a page or even close the tab/window. Simply, it will remain until it's removed. This is the ideal place to store information (`slices`) like
-    -   **User Consent agreement** (to not ask about the agreement of user consent each time a user opens the page)
-    -   **User CartUuid or ComparisonUuid** (to not lose an unauthenticated user's cart or comparison)
--   **Session Store** - Store for storing temporary information. All values saved in this store will be removed after reloading the page.
+As the name suggests, this store is persisted. That means that information saved in this store will stay in your browser (specifically in your local storage) after you reload the page or even close the tab/window. Simply, it will remain there until it's removed.
+
+### When should it be used?
+
+Persist store should be used if you need something to work across sessions, and not be influenced by user closing the browser. This could be storing the cart UUID, various long-term identifiers, etc. Mind that persist store is not available on the server (due to local storage not being available), so if you need to store something in a persistent manner, but need it on the server as well, you should use [cookie store](#cookie-store). If you do not need to persist the data you are working with, you should look into [session store](#session-store).
+
+### Setting the store name
+
+It is a good practice to rename the store when you start developing your project. You should pick a very specific name, which cannot be confused with another project. This is because if you jump between multiple projects based on this codebase, and develop all of them localy, they are essentially treated as the same application (they all use the same domain). This can result in the persisted store being all messed up in your local storage. This is exactly what causes all those weird errors which simply disappear once you open the application in incognito mode in your browser.
+
+Because of that, we suggest that the first thing you do with your persist store is a change like this:
+
+```diff
+- const STORE_NAME = 'shopsys-platform-persist-store';
++ const STORE_NAME = 'my-amazing-app-persist-store';
```

Comment:
isn't named `PERSIST_STORE_NAME` in the application?

Reply:
You are right. Thanks. Refactored.